tox.ini

```diff
@@ -33,6 +34,11 @@ deps = pre-commit
 commands_pre =
 commands = pre-commit run --all-files --show-diff-on-failure
 
+[testenv:mypy]
```

Comment:
We can add mypy to `pre-commit` config instead to be able to run it on `checkqa`.